### PR TITLE
ppc: disable 1G huge page test on ppc

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:2265

### CODE REVIEW

File: qemu/tests/cfg/hugepage_specify_node.cfg [hunk 1]
```diff
@@ -44,7 +44,7 @@
             #    Before start testing, please ensure your host OS support 1G hugepage.
             #    Please don't forget to update host kernel line to enable 1G hugepage
             #    Please ensure your host have enough memory to create guest memory.
-            no s390x
+            no s390x ppc64le ppc64
```
Comment (PaulYuuu) [inline qemu/tests/cfg/hugepage_specify_node.cfg:47]: IIRC, ppc supports 1G hagepage, why disable here?

Comment (MiriamDeng) [reply qemu/tests/cfg/hugepage_specify_node.cfg:47]: We didn't cover the test actually a long time.

Comment (PaulYuuu) [reply qemu/tests/cfg/hugepage_specify_node.cfg:47]: So this is what internal needs to consider, not upstream, as 1G hugepage is supported.

Comment (maxujun) [reply qemu/tests/cfg/hugepage_specify_node.cfg:47]: Make sense.

**Comment by MiriamDeng** [comment]
Close it due to the discussion

**Pull request closed without merging**